### ISSUE TITLE
Sublime-commands does not work

### ISSUE THREAD

**Issue by ShiBody** [facelessuser/RegReplace#77]
It seems that all RegReplace commands do not work on my mac. I tried to install RegReplace by Package control, but it did not show any installed success or failure information in the bottom of my sublime. Then I installed RegReplace by git clone, but I found all RegReplace commands like "RegReplace: Create New Regular Expression Rule" which are written in file Default.sublime-commands - RegReplace do not work. And the commands I wrote in Default.sublime-commands - User do not work as well. It seems the sentence " "command": "reg_replace" " does not work.

Mac 10.13.3
Sublime 3.0 Build 3143

**Comment by facelessuser** [comment]
You will need to post your configurations, or I will not be able to help since I have no idea what you are doing.  Also you would need to post any errors from the console.

**Comment by ShiBody** [comment]
I installed RegReplace with the following commands in console:
`cd Application\ Support/Sublime\ Text\ 3/Packages/`
`git clone https://github.com/facelessuser/RegReplace.git RegReplace`

Get results:
Cloning into 'RegReplace'...
remote: Counting objects: 1161, done.
remote: Compressing objects: 100% (41/41), done.
remote: Total 1161 (delta 14), reused 49 (delta 10), pack-reused 1106
Receiving objects: 100% (1161/1161), 661.77 KiB | 173.00 KiB/s, done.
Resolving deltas: 100% (668/668), done.

I reopen my sublime, press shift+command+P, input "regreplace".
Only show 4 commands in table list: 
- RegReplace: Documentation
- RegReplace: Settings
- RegReplace: User Rules
- RegReplace: Quick Start Guide

No other commands like "RegReplace: Create New Regular Expression Rule" in my table list. 

<img width="468" alt="screen shot 2018-02-24 at 16 04 33" src="https://user-images.githubusercontent.com/19902171/36627341-47f85978-197c-11e8-8b04-ed1a5bebc483.png">

For my custom configurations:
Default.sublime-commands - User : 
[
    {
        "caption": "RegReplace: remove_soh",
        "command": "reg_replace",
        "args":{"replacements": ["remove_soh"]},
        "find_only":false,
    }

]

reg_replace_rules.sublime-settings :
{
    "format": "3.0",
    "replacements": {
        "remove_soh": {
            "find": "soh",
            "replace": "",
            "greedy": true,
        }
}

Key bindings: 
[
	{
		"keys": ["super+shift+x"],
		"command": "reg_replace",
		"args": {"replacements": ["remove_soh"]}
	}
]

**Comment by facelessuser** [comment]
RegReplace is meant to be installed with package control.  If you install manually, you will have to install with additional steps due to dependencies: http://facelessuser.github.io/RegReplace/installation/#manual-installation.  I imagine you have console errors that you haven't posted.

**Comment by ShiBody** [comment]
Yes. I installed RegReplace according to the steps "Git Cloning" in the link you given, but I cannot see any console error.
<img width="715" alt="screen shot 2018-02-24 at 16 40 51" src="https://user-images.githubusercontent.com/19902171/36627688-5e60934c-1981-11e8-9418-5bfea8feaa57.png">

I tried to install it by Package Control, but it failed without any error message. And I did not see "Package Control Messages" which would pop out when installed successfully like other plugins.
